src/topper-web-sdk.ts

```diff
@@ -104,8 +104,8 @@ class TopperWebSdk {
       ...(isTopperSelfEmbed && { embed: 1 }),
       ...(this.eventHandlers &&
         Object.keys(this.eventHandlers).length && { events: Object.keys(this.eventHandlers).join(',') }),
-      ...(this.config.is_android_app && { is_android_app: 1 }),
-      ...(this.config.is_ios_app && { is_ios_app: 1 }),
+      ...(this.config.is_android_webview && { is_android_webview: 1 }),
```

Comment:
We should still support the legacy parameters and launch a minor version. Just don't keep them on the read me options table. Otherwise we could be breaking partner integrations.

Then we will release a major (in a month) removing them.